docs/Policies.md

```diff
@@ -0,0 +1,188 @@
+# Policies in EDC
+
+Purpose of this guide is to give an overview of what policies are and how they work in EDC.
+Policy model used internally is based on the [Open Digital Rights Language (ODRL)](https://www.w3.org/TR/odrl-model/) and going
+through it would help get a more complete picture. Nevertheless, this guide can be followed without it and the sections below
+provide a simplified overview of the main ODRL concepts.
+
+## What are policies?
+
+Conceptually, a policy is a non-empty collection of rules, which specify actions the consumer is _allowed_, _disallowed_
+or _required_ to perform on an Asset, optionally refined by a set of constraints.
+
+As indicated above, rules come in three types:
+- **Permissions**, which specify actions the consumer is allowed to perform on an asset
+    - _Example_: permission to use the asset, but only if the consumer is in the EU region
+- **Prohibitions**, which specify actions the consumer is not allowed to perform on an asset
+    - _Example_: prohibiting further distribution of the asset
+- **Obligations** (a.k.a. **Duties**), which specify actions the consumer is required to perform
+    - _Example_: obligation to delete the data after 30 days
+
+It is possible to link these entities together, forming more complex rule chains. For example, a _Prohibition_ can be
+extended with a set of _Obligations_, specifying actions that need to be performed in case the prohibition has been infringed.
```

Comment:
do we already support that case?

Reply:
We support the evaluation but no real mechanism in a sample